gatorgrade/output/output_functions.py

```diff
@@ -1,2 +1,30 @@
-# This class is used for storing the main functions requested from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output, description output, and colorization of text.
\ No newline at end of file
+''' This module is used for storing the main functions requested from the Github Issue Tracker for the output team.

```

Comment:
I had one more recommendation for these docstrings. Docstrings tend to be in triple double quotes  (""" """). Would you mind changing these single quotes to double quotes?